### ISSUE TITLE
img throws exception if not including *:// 

### ISSUE THREAD

**Issue by ventr1x** [daohoangson/flutter_widget_from_html#250]
Without wanting to insult anyone, but I've never seen a plugin causing issues on every single step I'm trying to take.
Especially if they're very simple ones.

I'm trying to build a custom Widget for img, as the implementation does not support things like FileImage (why not? it's an about 3 line snippet to make it work). Should be simple and easy enough, right?
Simple issue: If I put ANY string into src= that does not start with (http):// it throws an unpreventable exception because there is still internal parsing BEFORE the customWidgetBuilder is called (why? Sounds like a waste of performance); and again a VERY unstable one at that.

Just try it out:
` '<p><img src="hello"/></p>',`

Is throwing

```
The following NoSuchMethodError was thrown building HtmlWidget(dirty, state: _HtmlWidgetState#8daf3):
The getter 'length' was called on null.
Receiver: null
Tried calling: length
```

`'<p><img src="completelyuselessprotocol://hello"/></p>'`

Is "working".

Of course I can work around that by putting a random protocol in there and removing it again before building the image, but this random error wasted 3 hours of my life trying to find the error on my part.

**Comment by daohoangson** [comment]
What is your Flutter version? I'm assuming your are using the package version 0.5-rc? Actually I have just tested with both latest 0.4 and 0.5-rc and there is no error with this code:

<details>

<summary>test.dart</summary>

```dart
import 'package:flutter/material.dart';
import 'package:flutter_widget_from_html/flutter_widget_from_html.dart';

const kHtml = '''
Foo
<p><img src="hello"/></p>
Bar
''';

class HelloWorldCoreScreen extends StatelessWidget {
  @override
  Widget build(BuildContext context) => Scaffold(
        appBar: AppBar(
          title: Text('Issue 250'),
        ),
        body: HtmlWidget(kHtml),
      );
}
```

</details>

<img src="https://user-images.githubusercontent.com/239336/88422156-f77baf00-ce13-11ea-9710-ff1fdbe00aad.png" width="300" />

It would be helpful if you can provide the Dart code so I can reproduce the issue on my part. The test suite does have a few test cases that are very similar to your HTML so it should work.

Regarding `FileImage` support, it is indeed not working yet but adding `file://` protocol support shouldn't be too hard.

**Comment by ventr1x** [comment]
`0.5.0-rc.2020071301`


```
[✓] Flutter (Channel stable, v1.17.5, on Mac OS X 10.14.6 18G103, locale en-DE)
    • Flutter version 1.17.5 
    • Framework revision 8af6b2f038 (3 weeks ago), 2020-06-30 12:53:55 -0700
    • Engine revision ee76268252
    • Dart version 2.8.4
```


It seems to only happen when trying to build a custom widget:

```
import 'package:flutter/material.dart';
import 'package:flutter_widget_from_html/flutter_widget_from_html.dart';
import 'package:url_launcher/url_launcher.dart';

class ArticlesDetailsScreen extends StatefulWidget {

  ArticlesDetailsScreen();

  @override
  _ArticlesDetailsScreenState createState() => _ArticlesDetailsScreenState();
}

class _ArticlesDetailsScreenState extends State<ArticlesDetailsScreen> {
  @override
  void initState() {
    super.initState();
  }

  Widget _buildImg(e) {
    return Text("aaaa");
  }

  @override
  Widget build(BuildContext context) {
    var scaffold = Scaffold(
        resizeToAvoidBottomPadding: false,
        appBar: null,
        body: SingleChildScrollView(
        child: HtmlWidget(
          '<p><img src="hello"/></p>',
          customWidgetBuilder: (e) {
            if (e.localName == 'img') {
              return _buildImg(e);
            }

            return null;
          },
        ),
      )
    );
    return scaffold;
  }
}

```

**Comment by daohoangson** [comment]
Good news. This bug has been fixed in the `master` branch, the fix will be included in the next 0.5 release. Thank you for your patience.

**Comment by daohoangson** [comment]
The latest 0.5 version `0.5.0-rc.2020081901` has been released with the fix for this. Please try upgrading and let me know if it works for you.
